src/main/webapp/components/Btn.css

```diff
@@ -27,3 +27,114 @@
   font-size: 0.8rem;
   padding-left: 1px;
 }
+
+.cta-container {
+  float: left;
+}
+
+.cta-container .btn {
+  margin-right: 5px;
+}
+
+.cta-container .btn,
+.cta-container .btn:hover,
+.cta-container .btn:active,
+.cta-container .btn:focus {
+  outline: none;
+}
+
+.request-clarification {
+  display: inline-block;
+  position: relative;
+  height: 50px;
+  background: url('../../../../grails-app/assets/images/icon-live_help.svg') no-repeat center center;

```

Comment:
Do you know if this gets translated correctly with webpack? I would think that `/assets/...` should work here.